development/tools/jobs/tester/releases.go

```diff
@@ -0,0 +1,24 @@
+package tester
+
+// DO NOT EDIT. THIS FILE IS GENERATED
```

Comment:
As this file is generated, I'd rather remove the next comment, it is a bit misleading.